fr/orm/query-builder.rst

```diff
@@ -858,6 +858,24 @@ code SQL. En plus de ``join()`` vous pouvez utiliser ``rightJoin()``,
         ['Authors.promoted' => 'boolean', 'Authors.created' => 'datetime']
     );
 
+Notez que si vous définissez l'option ``quoteIdentifiers`` à ``true`` quand vous
+configurez votre ``Connection``, les conditions mettant en relation deux champs
+de tables différentes doivent être définies de cette manière ::
```

Comment:
here there must be no space before ::, otherwise the code nippet will be badly printed by sphinx